api/observability/v1/clusterlogforwarder_types.go

```diff
@@ -128,6 +129,13 @@ type CollectorSpec struct {
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Affinity"
 	Affinity *corev1.Affinity `json:"affinity,omitempty"`
+
+	// Define the maxUnavailable pod rollout strategy
```

Comment:
If not set will be used default value 100%, maybe need to add this information here in doc.